content/docs/concepts/metric_types.md

```diff
@@ -48,6 +48,10 @@ A _histogram_ samples observations (usually things like request durations or
 response sizes) and counts them in configurable buckets. It also provides a sum
 of all observed values.
 
+Since each observation bucket holds the total count of observations less than or
```

Comment:
A cumulative counter is, of course, something you expect of a counter.  To indicate that this is a **Cumulative Histogram**, and that that is actually a technical term (perhaps by linking to the Wikipedia page) would definitely clarify things.

It would be great if there were more detail explaining the manifold reasons _why_ a cumulative histogram is a good choice.  That was the one that Brian mentioned to me which made it super clear, but it'd be nice to know more about that, and so I shall now go read that Wikipedia article in the hope that I can learn more... :-)